pkg/api/titond_api.go

```diff
@@ -27,3 +28,33 @@ func (t *TitondAPI) CreateNetwork(data *model.Network) *model.Network {
 
 	return result
 }
+
+func (t *TitondAPI) CreateL2Geth(data *model.Resource) {
+	// TODO : deal with DB
+	t.db.CreateResource()
+
+	namespace := "default"
+	createL2Geth(t.k8s, namespace)
```

Comment:
Please call this using goroutine. and we can update status of l2geth to true in database in `createL2geth`. And it seems great using receiver function as @nguyenzung say. `func (t *TitondAPI) createL2Geth(namespace string)`

Reply:
fix it! Thank you!